reflex/components/markdown/markdown.py

```diff
@@ -199,7 +201,15 @@ def get_component(self, tag: str, **props) -> Component:
             raise ValueError(f"No markdown component found for tag: {tag}.")
 
         special_props = [_PROPS_IN_TAG]
-        children = [_CHILDREN]
+        children = [
+            _CHILDREN
+            if tag != "codeblock"
+            else ternary_operation(
```

Comment:
can there be a clarifying case on why children of a codeblock be a list? and why is it an empty str as opposed to a new line?

Reply:
Possibly a comment should improve this

Reply:
I think I had the `join` use an empty string instead of a newline just so these delimiters are controlled in the code, but I am fine with having this as a newline  since it doesn't make so much of a difference